.github/workflows/go.yml

```
@@ -38,7 +38,9 @@ jobs:
     - name: Download Dependencies
       run: |
         go mod download
-        cd tools && go mod download
+        cd tools && go mod download && cd -
+        cd benchmarks && go mod download && cd -
+        cd zapgrpc/internal/test && go mod download && cd -
```

We can use subshells to avoid cding back.

```suggestion
        (cd tools && go mod download)
        (cd benchmarks && go mod download)
        (cd zapgrpc/internal/test && go mod download)
```

---

good call!